bower.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "axe-core",
   "version": "4.6.3",
+  "deprecated": true,
```

Comment:
This is not actually a supported prop. I put something in so that when we get to 5.0 and search for deprecated things to remove, we'll find this.